dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -65,7 +73,7 @@ def cleanData(cls, data, model, fieldMap, fieldOptions):
             if "pad_to_length" in options:
                 # pad to specified length
                 data['{}'.format(col)] = data['{}'.format(col)].apply(
-                    lambda x: cls.padFunction(x, options['pad_to_length']))
+                    cls.padFunction, args=(options['pad_to_length'],))
```

Comment:
This threw me for a loop; I expected `args` to be the _first_ arguments to the function rather than the last. Good to know!